mtbf_driver/mtbf.py

```diff
@@ -145,6 +146,9 @@ def start_gaiatest(self):
                 except NoSectionError as e:
                     self.logger.error(e)
                     continue
+                except DMError as e:
+                    self.logger.error(e)
```

Comment:
Do we need to have a not about which exception we catch? Like have a note of NoSectionError type or DMError type?